packages/client/test/integration/dataunion/binanceAdapter.test.ts

```diff
@@ -0,0 +1,60 @@
+import { BigNumber, Contract, providers, Wallet } from 'ethers'
```

Comment:
Lots of linter errors, please fix lint issues (run `npm run eslint`)

Reply:
at least IDEA and VScode have "fix all eslint problems" functionality that mostly fixes them immediately.

Reply:
ok Ill try

Reply:
👍